test/components/ResultTypes.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import {
+    CourseItemResult,
+    GenericPageResult
+} from './../../src/components/ResultTypes';
+
+const id = 'TEST_ID';
+const description = 'TEST_DESCRIPTION';
+const pageName = 'TEST_PAGENAME';
+const siteName = 'TEST_SITENAME';
+const links = ['TEST_LINK1', 'TEST_LINK2'];
+
+describe('CourseItemResult', () => {
```

Comment:
just wondering, is it standard practice to unit test objects like this that have no special functions (ie these result types that exist solely to hold data fields)?

Reply:
Nope! Haha. This really doesn't need to be tested, I just wanted good coverage numbers ;)